server/blob/blob-cassandra/src/test/java/org/apache/james/blob/cassandra/cache/CachedBlobStoreTest.java

```diff
@@ -300,12 +301,15 @@ public void readBlobStoreCacheWithNoneDefaultBucketNameShouldNotImpact() {
                 assertThat(metricFactory.executionTimesFor(BLOBSTORE_CACHED_LATENCY_METRIC_NAME))
                     .describedAs(BLOBSTORE_CACHED_LATENCY_METRIC_NAME)
                     .hasSize(0);
+                assertThat(metricFactory.executionTimesFor(BLOBSTORE_BACKEND_LATENCY_METRIC_NAME))
```

Comment:
Separate cached and backend latency tests

Reply:
what i mean it: record `cached_latency` but not `backend_latency` at the same time

Reply:
And maybe i can add more tests for record `backend_latency`

Reply:
Either use soft assertions or write separated test cases (I think René asked for a separate test)

Reply:
This test should be in `soft assertions`, my bad when copy.Also i will add a separate test for new case.